doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,281 @@
+17) [Location](#17-location)                                                 
+18) [Ngsiv2-AttrsFormat](#18-ngsiv2-attrsformat)                                    
+19) [Fiware-Service](#19-fiware-service)
+20) [Fiware-Servicepath](#20-fiware-servicepath)
+21) [Fiware-Total-Count](#21-fiware-total-count)
+22) [Fiware-Correlator](#22-fiware-correlator)                                     
+
+23) [X-Auth-Token](#23-x-auth-token)
+
+
+
+# HTTP headers used by Orion with their description
+
+
+
+## 1. Accept
+
+Accept HTTP header is used in incoming HTTP requests received by Orion. 
+Accept HTTP header is used in the request to specify which MIME type the client accepts. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`. 
+For example in request URL if we use:
+	
+    curl ... -H 'Accept: application/json'
+		 
+On this request we'll get the response data in well defined JSON format.
+
+The API response payloads in this specification are based on `application/json` and  (for attribute value 
+type operation) `text/plain` MIME types. Clients issuing HTTP requests with accept types different 
+than those will get a `406 Not Acceptable` error.
+
+[Top](#top)
+
+## 2. Content-Length
+
+This header is used by both requests and responses.
+Content-length HTTP header is a mandatory header in Orion responses.
+It defines the length of the request and response body in bytes.
+Orion Context Broker expects always a Content-Length header in all client requests, otherwise the client will receive a `411 Length Required` response. 
+This is due to the way the underlying HTTP library (libmicrohttpd) works.
+	
+    Content-Length: 34	
+
+[Top](#top)
+ 
+## 3. Content-Type 
+
+It is used to specify the MIME type of the request or response. Typically its value is `application/json` (for JSON MIME type) although some operations in the NGSIv2 API also allow `text/plain`.
+
+    curl ... -H 'Content-Type: application/json'
+
+This request will send data in well defined JSON format.
+
+[Top](#top)
+ 
+## 4. Connection
+ 
+ This header is used by both requests and responses.
+ Orion is not currently doing anything with this header (apart from checking if its value is `close` to print a log trace).
+	
+    Connection: Keep-Alive
+    	
+[Top](#top)	
+  
+## 5. Expect
+
+Expect HTTP header is used in incoming HTTP requests received by Orion.
+It is used when sending a large request body, on response Orion sends only `Expect:` i.e. blank, if it is able to handle the request. For more details, follow general documentation of Expect HTTP header [Link1](https://flaviocopes.com/http-request-headers/), [Link2](https://tools.ietf.org/html/rfc7231#section-5.1.1) or [Link3](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields).
+
+
+    Expect:
+	
+[Top](#top)	
+	
+## 6. Host
+
+This HTTP header is used in Orion responses.
```

Comment:
As far as I understand this is not correct. Orion doesn't include the Host header in responses. If I'm wrong, please tell me in which part of the code is done (I could be missing something...). Otherwise, please remove L94.

Reply:
Fixed in 512f9cd